changelog/v1.19.0-rc4/lambda-persist-request-details.yaml

```diff
@@ -0,0 +1,8 @@
+changelog:
+- type: NEW_FEATURE
+  issueLink: https://github.com/solo-io/gloo/issues/5560
+  resolvesIssue: false
+  description: >
+    Adds lambda option to persist the original request info that otherwise gets lost.
+    Set as an option to avoid leaking potentially sensitive path info by default.
```

Comment:
include rationale for false by default as well for the mesh case, following example of xff header?